src/components/Form/SetAclPermsDocContainerForm.jsx

```diff
@@ -56,37 +50,21 @@ const SetAclPermsDocContainerForm = () => {
     try {
       await setDocContainerAclPermission(session, permissions, podUrl, webIdToSetPermsTo);
 
-      runNotification(
-        `${permissions.read ? 'Give' : 'Revoke'} permission to ${
-          permissionState.podUrlToSetPermissionsTo
-        } for Documents Container.`,
-        5,
-        state,
-        dispatch
-      );
       addNotification(
         'success',
         `${permissions.read ? 'Gave' : 'Revoked'} permission to ${
           permissionState.podUrlToSetPermissionsTo
         } for Documents Container.`
       );
     } catch (error) {
-      runNotification('Failed to set permissions. Reason: File not found.', 5, state, dispatch);
       addNotification('error', 'Failed to set permissions. Reason: File not found.');
     } finally {
-      setTimeout(() => {
-        clearInputFields();
-      }, 3000);
+      setTimeout(() => {}, 3000);
```

Comment:
don't think this does anything

Reply:
This currently does nothing. I can delete it unless we are going with Ka Hung's comment of keeping them for a future setProcessing(false), which I assume will be part of a different PR?

Reply:
If it doesnt do anything, we can get rid of it. It's in the git history, so we can restore it later if we need

Reply:
I've addressed Ka Hung's comment and now the timeouts are being used to switch the processing state.